db/migrate/20240225160909_create_page_texts.rb

```diff
@@ -1,4 +1,4 @@
-class CreatePageTexts < ActiveRecord::Migration[7.0]
+class CreateCustomPages < ActiveRecord::Migration[7.0]
```

Comment:
Why class name is different than file name? I am not sure if this is good way to go

Reply:
Yes it should be the same. This was fixed in main branch a few days ago as it was an error.

Reply:
Thank you